test/e2e/operator/validate/validate.go

```diff
@@ -281,7 +281,7 @@ StatefulSet:` + defaultsApps + `
 // In addition, it fills in default values in the expected spec if they are missing
 // before comparing against the actual value. This makes it possible to
 // compare the on-disk YAML files which are typically not complete against
-// objects frome the apiserver which have all defaults filled in.
+// objects from the apiserver which have all defaults filled in.
```

Comment:
Please submit whitespace, punctuation and spelling fixes in one separate PR.

Reply:
This change went in as part of a different commit.